CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ cmake_minimum_required(VERSION 3.11)
 include(CPack)
 
 project(dlite
-  VERSION 0.2.2-alpha
+  VERSION 0.2.2.1
```

Comment:
Doesn't semantic versioning require hypen instead of dot between patch version number and pre-release version? See point 9 on https://semver.org/